examples/python/interface.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+"""
+This module is an example python-based testing interface.  It uses the
+``requests`` package to make REST API calls to the test case container,
+which mus already be running.  A controller is tested, which is
+imported from a different module.
+
+"""
+
+# GENERAL PACKAGE IMPORT
+# ----------------------
+import requests
+import time
+import numpy as np
+from examples.python.custom_kpi import custom_kpi_calculator as kpicalculation
+import json,collections
+import pandas as pd
+import importlib
+
+def control_test(config):
+    '''Run test case.
+
+    Parameters
+    ----------
+    plot : bool, optional
+        True to plot timeseries results.
+        Default is False.
+    config : string, dict

```

Comment:
I think it'd be better to remove the ``config`` construct and instead just have keyword parameters like ``step``, ``length``, etc. with proper descriptions of each in the docstring.

Reply:
I am not sure if it is extensible. How about leaving those details to the configuration files?

Reply:
I'm sorry I'm not sure what you mean by extensible.  My suggestion is to have:
```
def control_test(control_module, length, step, customized_kpi_config=None, prediction_config=None):
```
or if not, then to define each expected keyword of the argument ``config`` in the docstring, so it is clear what each configuration parameter represents.

I also suggest to change ``prediction_config`` to ``forecast_config`` so as to be consistent with the API call it refers to so as to improve readability.

Reply:
As suggested, we changed the input arguments of control_test